### PR TITLE
Update to documentation for python version and allow for sites drush …

### PR DESCRIPTION
**Pull request by sherakama**
…aliases to be configurable.

### CODE REVIEW

**Comment by jbickar** [comment]
Also, can you send me your `aliases.drushrc.php` so I can test this? Mine are clearly set to still use `ds_` and `dp_` (because both prefixes exist on sites-uat).

**Comment by sherakama** [comment]
Hey sorry, I didn't put in a body on this PR as it was late last night but thanks for looking it over.

You can see my alias file here: https://gist.github.com/sherakama/7d97141d7bd43e933cd3#file-sites-aliases-drushrc-php-L20

I use the syntax `drush @sse.shortname` & `drush @ppl.shortname` instead of the fold directory as I am lazy and don't like to type.

I added the drush prefix variable so that I may avoid altering my drush aliases and the rest of the scripts that are reliant on them but left it so that by default your syntax still works. Ideally the aliases would be bundled in this PR like the acsf ones.

**Comment by sherakama** [comment]
I also couldn't get `validate_certs: false` working and was unsure where to put that setting so I put it everywhere and still no dice.

**Comment by jbickar** [comment]
I think we decided not to bundle the `@sse` aliases so as not to stomp on anyone's alias file, but it would make sense to include them and namespace them properly.

File: group_vars/all.yml [hunk 1]
```diff
@@ -33,3 +33,6 @@ drush_environment: ""
 # default, in order to avoid "undefined variable" error messages. It can be
 # set on a per-site basis to "TRUE" in the inventory file. It is case-sensitive.
 ignore_updb_errors: ""
+
+# If you have something different than ds_ or dp_
+sites_drush_alias: "default"
```
Comment (jbickar) [inline group_vars/all.yml]: This should be in `migration_vars.yml`, which is not tracked in version control. So for the PR you would put it in `default.migration_vars.yml`

Comment (sherakama) [reply group_vars/all.yml]: Thanks, I'm just getting my feet wet with ansible.

**Comment by jbickar** [comment]
Hm. I'm not sure how I feel about this. As pointed out, we should bundle the drush aliases with the project. _Or_, we should refactor the `sites_prefix` variable so that it can be overridden by the user.

However, that is probably more work than we have time for now. So I think I will merge this and create a ticket to provide drush aliases, and remove these new tasks as part of that ticket.

**Comment by sherakama** [comment]
Who else is likely to run these other than us? Punt.

**Comment by jbickar** [comment]
The new dev we're hiring. No names shall be mentioned, as this is a public repo.

**Comment by jbickar** [comment]
It is an amusing situation for you to be on Team Punt and me to be on Team Ideological Purity, for once 😆

**Comment by sherakama** [comment]
This migration is far from ideological purity. Sometimes, one must play down in the muck to get a dirty job done.

File: roles/download-site/tasks/main.yml [hunk 2]
```diff
@@ -29,6 +29,16 @@
...
 - name: Download copy of database from sites
   shell: "drush @{{ server_alias }}.{{ site_prefix }}_{{ inventory_hostname }} sql-dump --structure-tables-key=common > /tmp/{{ inventory_hostname }}/dbdump.sql"
+  when: sites_drush_prefix == 'default'
+
+- name: Set sites drush alias
+  set_fact:
+    sites_drush_alias: "{{ server_alias }}.{{ sites_drush_prefix }}{{ inventory_hostname }}"
+  when: sites_drush_prefix != 'default'
+
+- name: Download copy of database from sites
```
Comment (jbickar) [inline roles/download-site/tasks/main.yml]: Shoot, this needs a distinct name from the other task, due to `--start-at-task`

**Pull request merged**